assert_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+
+package gosnowflake
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func assertNilF(t *testing.T, actual any, descriptions ...string) {
+	errMsg := validateNil(actual, descriptions...)
+	if errMsg != "" {
```

Comment:
I can see a lot of duplications here. Maybe we can write it as:
```
errMsg := ...
fatalOnEmpty(t, errMsg)?
```

Reply:
changed, even in 1 liner